pulsar-client/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java

```diff
@@ -309,4 +309,13 @@ ClientBuilder authentication(String authPluginClassName, Map<String, String> aut
      * @param unit time unit for {@code statsInterval}
      */
     ClientBuilder keepAliveInterval(int keepAliveIntervalSeconds, TimeUnit unit);
+
+    /**
+     * Set the duration of time to wait for a connection to a broker to be established. If the duration
+     * passes without a response from the broker, the connection attempt is dropped.
+     *
+     * @param duration the duration to wait
+     * @param unit the time unit in which the duration is defined
```

Comment:
Can you add `@since` tag? so people will know when this method was added in javadoc.

Reply:
Will add